zerver/openapi/zulip.yaml

```diff
@@ -1225,657 +1263,3511 @@ paths:
+                                  type: object
+                                  description: |
+                                    Object containing user id and email of the sender i.e
+                                    the user who has started typing.
+                                  properties:
+                                    user_id:
+                                      type: integer
+                                      description: |
+                                        The id of the user.
+                                    email:
+                                      type: string
+                                      description: |
+                                        The email of the user.
+                                recipients:
+                                  type: array
+                                  description: |
+                                    Array of dictionaries where each dictionary contains the
+                                    email and user_id of a single user who will receive the
+                                    typed message.
+                                  items:
+                                    type: object
+                                    additionalProperties: false
+                                    description: |
+                                      Object containing user id and email of a recipient.
+                                    properties:
+                                      user_id:
+                                        type: integer
+                                        description: |
+                                          The id of the user.
+                                      email:
+                                        type: string
+                                        description: |
+                                          The email of the user.
+                              example:
+                                {
+                                  "type": "typing",
+                                  "op": "start",
+                                  "sender":
+                                    {
+                                      "user_id": 10,
+                                      "email": "user10@zulip.testserver",
+                                    },
+                                  "recipients":
+                                    [
+                                      {
+                                        "user_id": 8,
+                                        "email": "user8@zulip.testserver",
+                                      },
+                                      {
+                                        "user_id": 10,
+                                        "email": "user10@zulip.testserver",
+                                      },
+                                    ],
+                                  "id": 0,
```

Comment:
Is `user10` being repeated as both sender and recipient a normal payload for this?

In general, how are you getting these event payloads?  From the test suite or hand-writing them?

Reply:
I mostly copy the examples generated from test_events.

Reply:
OK, makes sense.

Reply:
It would be good for the documentation in this example to clarify whether the sender will always appear in the recipients list.